src/main/java/reposense/model/AuthorConfiguration.java

```diff
@@ -145,10 +145,27 @@ public void addAuthor(Author author, List<String> ignoreGlobList) {
     public void removeIgnoredAuthors(List<String> ignoredAuthorsList) {
         for (String author : ignoredAuthorsList) {
             if (authorEmailsAndAliasesMap.containsKey(author)) {
-                authorList.remove(authorEmailsAndAliasesMap.get(author));
+                removeAuthor(author);
             }
         }
-        resetAuthorInformation();
+    }
+
+    /**
+     * Removes all information of the {@code author} from the configs
+     * @param author Can be an author's git ID, email, or alias
```

Comment:
Perhaps we should state that `author` must be in the `authorEmailsAndAliasesMap` as a pre-condition

Reply:
Ya, can include.